content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx

```diff
@@ -13,6 +13,8 @@ Some development-focused commands for relational databases of Prisma Migrate use
 
 The shadow database is **created and deleted automatically**\* each time you run a development-focused command and is primarily used to **detect problems** such as schema drift.
 
+[`migrate diff` command](/reference/api-reference/command-reference#migrate-diff) also requires a shadow database when diffing against a local `migrations` directory.
```

Comment:
Can we list the arguments that is the case for?

Reply:
Only `--to/from-migrations` like listed in the reference.

Reply:
```suggestion
[`migrate diff` command](/reference/api-reference/command-reference#migrate-diff) also requires a shadow database when diffing against a local `migrations` directory with `--to/from-migrations`.
```